src/CommunityToolkit.Maui.UnitTests/Converters/ColorToColorConverter_Tests.cs

```diff
@@ -0,0 +1,222 @@
+	{
+		var converter = new ColorToColorForTextConverter();
+
+		Assert.Throws<ArgumentException>(() => converter.Convert(value, typeof(Color), null, null));
+	}
+
+	[Fact]
+	public void ColorToColorForTextConverterConvertFromInvalidArgumentsTest()
+	{
+		var converter = new ColorToColorForTextConverter();
+
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+		Assert.Throws<ArgumentNullException>(() => converter.ConvertFrom(null));
+		Assert.Throws<ArgumentNullException>(() => converter.ConvertFrom(default));
+#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
+	}
+
+	[Theory]
+	[MemberData(nameof(ColorToGrayScaleColorData))]
+	public void ColorToGrayScaleColorConverterValidArgumentsTest(Color initialColor, Color expectedColor)
+	{
+		var converter = new ColorToGrayScaleColorConverter();
+
+		var convertedColor = converter.Convert(initialColor, typeof(Color), null, null);
+		var convertedColorFrom = converter.ConvertFrom(initialColor);
+
+		Assert.Equal(expectedColor, convertedColor);
+		Assert.Equal(expectedColor, convertedColorFrom);
+	}
+
+	[Theory]
+	[InlineData(null)]
+	[InlineData(2)]
+	[InlineData('c')]
+	[InlineData(true)]
+	public void ColorToGrayScaleColorConverterConvertInvalidArgumentsTest(object? value)
+	{
+		var converter = new ColorToGrayScaleColorConverter();
+
+		Assert.Throws<ArgumentException>(() => converter.Convert(value, typeof(Color), null, null));
+	}
+
+	[Fact]
+	public void ColorToGrayScaleColorConverterConvertFromInvalidArgumentsTest()
+	{
+		var converter = new ColorToGrayScaleColorConverter();
+
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+		Assert.Throws<ArgumentNullException>(() => converter.ConvertFrom(null));
+		Assert.Throws<ArgumentNullException>(() => converter.ConvertFrom(default));
+#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
+	}
+
+	[Theory]
+	[MemberData(nameof(ColorToInverseColorConverterData))]
+	public void ColorToInverseColorConverterConverterValidArgumentsTest(Color initialColor, Color expectedColor)
+	{
+		var converter = new ColorToInverseColorConverter();
+
+		var convertedColor = converter.Convert(initialColor, typeof(Color), null, null);
+		var convertedColorFrom = converter.ConvertFrom(initialColor);
+
+		Assert.Equal(expectedColor, convertedColor);
+		Assert.Equal(expectedColor, convertedColorFrom);
+	}
+
+	[Theory]
+	[InlineData(null)]
+	[InlineData(2)]
+	[InlineData('c')]
+	[InlineData(true)]
+	public void ColorToInverseColorConverterConvertInvalidArgumentsTest(object? value)
+	{
+		var converter = new ColorToInverseColorConverter();
+
+		Assert.Throws<ArgumentException>(() => converter.Convert(value, typeof(Color), null, null));
+	}
+
+	[Fact]
+	public void ColorToInverseColorConverterConvertFromInvalidArgumentsTest()
+	{
+		var converter = new ColorToInverseColorConverter();
```

Comment:
probably you can combine these tests and path converter as parameter

Reply:
True.

I think I'll leave them separate since it'll provide a better error message if/when one of them fails. Plus, I'm also being a little lazy since I already wrote the code 😊